apps/admin/src/components/prompts/standard/submit-prompt.vue

```diff
@@ -1,16 +1,74 @@
 <template>
-  <v-tab-item key="options" value="options"></v-tab-item>
+  <v-tab-item key="options" value="options">
+    <v-card-title>"Final Review Options"</v-card-title>
+    <v-card-subtitle> "Choose the option you want for the final review page." </v-card-subtitle>
+    <v-card-text>
+      <v-combobox
+        v-for="(value, key) in review"
+        :key="key"
+        v-model="localReview[key]"
+        :items="[false, 'checkbox', 'scroll']"
+        :label="key"
+        @change="updateReviewOptions(key, $event)"
+      >
+      </v-combobox>
+    </v-card-text>
+  </v-tab-item>
 </template>
 
 <script lang="ts">
+import type { PropType } from 'vue';
 import { defineComponent } from 'vue';
 
+import type { Prompts } from '@intake24/common/prompts';
+
 import { basePrompt } from '../partials';
 
 export default defineComponent({
   name: 'SubmitPrompt',
 
   mixins: [basePrompt],
+
+  props: {
+    desktopReview: {
+      type: [Boolean, String] as PropType<Prompts['submit-prompt']['desktopReview']>,
+      required: true,
+    },
+    mobileReview: {
+      type: [Boolean, String] as PropType<Prompts['submit-prompt']['mobileReview']>,
+      required: true,
+    },
+    review: {
+      type: Object as PropType<Prompts['submit-prompt']['review']>,
+    },
+  },
+
+  data() {
+    return {
+      localReview: { ...this.review },
```

Comment:
Istead of keeping local state, you could do either the option we discussed or use a computed with setter
```
computed({
get() {
return this.review;
}
set(value) {
this.update('review', value);
}
})
```
I think that should work, watchers tend to complicate it

Reply:
Thanks for the suggestion. I have changed it to the option we discussed. 
```
<v-combobox
        v-for="key in Object.keys(review)"
        :key="key"
        :label="key"
        :value="review[key]"
        :items="[false, 'checkbox', 'scroll']"
        @change="update('review', {...review, [key]: $event})"
>
```